asyncio port scanner.py

```diff
@@ -0,0 +1,31 @@
+import time
+import asyncio
+
+
+async def connect_socket(ip: str, port: int):
+    """
+    The function try to connect a given socket to a given ip and port. If not
+    succeeded it raise an exception and print a massage.
+    :param ip: The ip for the socket to connect.
```

Comment:
Add space between summary and type docs.
```suggestion

    :param ip: The ip for the socket to connect.
```